### PR TITLE
Split models.rs into modules

### LINKED ISSUE #30: Enhance crate structure
The file `models.rs` is getting quite big, so I'd like to split it into smaller sub-modules.

What would be the best structure ? In thinking about
```
lib
├ issues
│ ├ api
│ ├ models
│ └ params
├ orgs
│ ├ api
│ ├ models
│ └ params
├ pulls
│ ├ api
│ ├ models
│ └ params
└ …
```
and
```
lib
├ api
│ ├ issues
│ ├ orgs
│ ├ pulls
│ └ …
├ models
│ ├ issues
│ ├ orgs
│ ├ pulls
│ └ …
└ params
  ├ issues
  ├ orgs
  ├ pulls
  └ …
```

### PR DESCRIPTION
**Pull request by kellda**
Fixes #30 

I made only one PR, but with one commit per new module.

### CODE REVIEW

**Comment by kellda** [comment]
@XAMPPRocky can you review this ?

File: src/models.rs [hunk 1]
```diff
@@ -947,70 +526,16 @@ pub struct InstallationToken {
...
 pub struct PublicKey {
     pub key_id: String,
     pub key: String,
 }
+
+pub mod issues;
+pub mod orgs;
+pub mod pulls;
+pub mod repos;
+pub mod teams;
```
Comment (XAMPPRocky) [inline src/models.rs:541]: Can you put these at the top of the file rather than the bottom?

**Comment by kellda** [comment]
Thank you, I've corrected it

**Comment by XAMPPRocky** [comment]
Thank you again for your PR!

**Pull request merged**